lib/scenic/adapters/postgres.rb

```diff
@@ -7,6 +7,17 @@ module Scenic
   # Scenic ships with a Postgres adapter only but can be extended with
   # additional adapters. The {Adapters::Postgres} adapter provides the
   # interface.
+  module Postgres
+    # Custom error classes for PostgreSQL specific errors
+    #
+    # These error classes are for more descriptive errors when users attempt
+    # to use materialized view features not supported by their current version.
+    class MaterializedViewsNotSupportedError < StandardError
```

Comment:
What do you think about having both of these supply a default message? They could do so either with:

```ruby
def initialize
  super("Here is the default message")
end
```

or:

```ruby
def message
  "here is the default message"
end
```

I think it'd be nice to say "Materialized views require Postgres 9.3 or newer" or "Concurrent materialized view refreshes require Postgres 9.4 or newer" to give some more specifics.